src/Pages/CreateMembers.jsx

```diff
@@ -4,30 +4,92 @@ import Box from "@mui/material/Box";
 import TextField from "@mui/material/TextField";
 
 export default function (props) {
+  const { API_URL } = props;
+
+  const [memberToCreate, setMemberToCreate] = useState({
+    userName: "",
+    membershipType: "",
+    membershipStatus: "",
+    picture: "smth",
+    firstName: "",
+    lastname: "",
+    houseNumber: 0,
+    streetName: "",
+    city: "",
+    postcode: "",
+    country: "",
+  });
```

Comment:
I can see what you're doing here but it's a little messy. I think adding some more state to represent `profile` and `address` would be great here:
`const [profileToCreate, setProfileToCreate] = useState({})`

Then when you get to the handle submit, you can change the `const newMember` to:

```
const newMember = {
...memberToCreate,
profile: {
  ...profileToCreate
  address: {
    ...addressToCreate
  }
}
}
```